osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/FindEventsCommandExecutor.java

```diff
@@ -168,6 +170,10 @@ private SelectiveAccessDescription getSelectiveAccessDescription(
      * value to determine which elements from the buffered array should be
      * retrieved.
      */
+
+    final DateTime convertedBeginDateTime = toDateTime(beginDateTime.toDate(), device);
```

Comment:
I would add a method to the converter that converts (DateTime, timezone) to DateTime
The name converted as prefix confuses me, as it is non-functional

Reply:
Added the method.